docs/man_pages/project/configuration/web-view-set.md

```diff
@@ -0,0 +1,42 @@
+web-view set
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder web-view set <Platform> <WebViewName>`
```

Comment:
Again - will users be able to run this command for Windows Phone? 
If not, it's better to go with the following format
`Android | `$ appbuilder web-view set android <Web View Name>`
`iOS | `$ appbuilder web-view set ios <Web View Name>`